### ISSUE TITLE
Segmentation fault with `sprite exportall` on CSG1.DAT

### ISSUE THREAD

**Issue by colejohnson66** [OpenRCT2/OpenRCT2#14522]
**OS:** Arch Linux
**Commit/Build:** 594b08c911a739070ee161359b559e7772db325c

Running `openrct2 sprite exportall CSG1DAT ~/csg1` causes a segmentation fault. It works with RCT2's g1.dat however. The fault occurs in `BlitPixel<1>` according to GDB. I believe the issue is `SpriteFile::Open` not parsing the file correctly, but I could be wrong.

**Steps to reproduce:**
1. Run the above command (replacing CSG1DAT with it's actual path)
2. Observe a segmentation fault.

**Comment by Gymnasiast** [comment]
csg1.dat is not of the same format as g1.dat, so it's not supposed to work. Although it should still result in an error message, rather than a segfault.

 It is possible to make a csg1.dat into a g1.dat-compatible file by prepending it with a specifically-written header and the contents of csg1i.dat.

**Comment by colejohnson66** [comment]
Yes, that worked. In addition to the segfault, the help screen that's printed to the console makes no indication that it's "RCT2 only", which is what led me to do such a thing.

**Comment by Gymnasiast** [comment]
That should be updated too, then.
